tests/test_locality_NeighborQuery.py

```diff
@@ -475,6 +479,19 @@ def test_throws(self):
         with self.assertRaises(RuntimeError):
             list(aq.queryBall(points, L))
 
+    def test_chaining(self):
+        N = 2500
```

Comment:
In case this is slow you could drop `N` for the unit test. If not, then don't worry about it.